### ISSUE TITLE
Compiler error:  object xml is not a member of package scala

### ISSUE THREAD

**Issue by jasoneales** [gatling/gatling#4075]
I upgraded from Gatling 3.4.1 to Gatling 3.5.1 

**Steps**

- Downloaded the "package" and extracted it to a directory
- Copied the "user-files" from 3.4.1 into the "user-files" of 3.5.1

When I run the "gatling.bat" file I get the following error

`10:31:01.524 [ERROR] i.g.c.ZincCompiler$ - C:/CodeRepos/test-NonFunctional/TestingParrellel/user-files/simulations/Agent/Agent.scala:10:14: object xml is not a member of package scala
did you mean Nil?
import scala.xml.XML`

I compared the "lib" folders of 3.4.1 and 3.5.1 and found that the "scala-xml_2.12-1.0.6.jar" is no longer there.

**Comment by slandelle** [comment]
Any library in the classpath that not required by our DSL is an implementation detail

Similarly, anyone using build tools such as maven, gradle or sbt and requiring some libraries for their own code to compile shouldn't rely on those to be transitively pulled but should explicitly import them.

Here, `scala-xml` was indeed a transitive dependency: we didn't use it ourselves, but it used to be used by one of the library we do use but that's not longer the case.

In short: you're in charge out providing the extra libraries you need for your own custom code to compile. You can download the [scala-xl_2.13 jar on maven central](https://search.maven.org/artifact/org.scala-lang.modules/scala-xml_2.13/1.3.0/bundle).
